### PR TITLE
New package: pdsh-2.33

### PR DESCRIPTION
**Pull request by knusbaum**

### CODE REVIEW

**Comment by knusbaum** [comment]
Fails to build on musl because lack of rresvport and a few others.

File: srcpkgs/pdsh/template [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Template file for 'pdsh'
+pkgname=pdsh
+version=2.33
+revision=1
+archs="x86_64 i686 aarch64 armv7l"
```
Comment (Chocimier) [inline srcpkgs/pdsh/template]: Use ~*-musl to disable musl.

File: srcpkgs/pdsh/template [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+desc_option_with_xcpu="Include support for the xcpu remote module"
+desc_option_with_netgroup="allows pdsh to use standard netgroup entries to build lists of target hosts."
+desc_option_with_readline="Include readline support"
+
+pre_configure() {
+	./bootstrap
+}
+
+post_install() {
+	rm ${DESTDIR}/usr/lib/pdsh/*.a
```
Comment (Chocimier) [inline srcpkgs/pdsh/template]: No configure parameter to not generate those?

Comment (knusbaum) [reply srcpkgs/pdsh/template]: Fixed.

File: srcpkgs/pdsh/template [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ $(vopt_if with_readline readline-devel)
+"
+short_desc="High performance, parallel remote shell utility"
+maintainer="Kyle Nusbaum <knusbaum+void@sdf.org>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/chaos/pdsh"
+distfiles="https://github.com/chaos/pdsh/archive/pdsh-${version}.tar.gz"
+checksum=c026ccf06e4f1c6c52414282a865e45362405aa09f7d09fa86aaec98f6413554
+
+build_options="with_rsh with_exec with_ssh with_mrsh with_xcpu with_netgroup with_readline"
```
[the remark this answers is not in the document]
Comment (knusbaum) [reply srcpkgs/pdsh/template]: Fixed.

File: srcpkgs/pdsh/template [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+configure_args="
+ $(vopt_if with_rsh --with-rsh)
+ $(vopt_if with_exec --with-exec)
+ $(vopt_if with_ssh --with-ssh)
+ $(vopt_if with_mrsh --with-mrsh)
+ $(vopt_if with_xcpu --with-xcpu)
+ $(vopt_if with_netgroup --with-netgroup)
+ $(vopt_if with_readline --with-readline)
+"
+hostmakedepends="automake autoconf libtool git gettext"
```
[the remark this answers is not in the document]
Comment (knusbaum) [reply srcpkgs/pdsh/template]: Removed.

File: srcpkgs/pdsh/template (hunk 1 above)
Comment (knusbaum) [reply srcpkgs/pdsh/template]: Fixed.

File: srcpkgs/pdsh/template [hunk 5]
```diff
@@ -0,0 +1,45 @@
+# Template file for 'pdsh'
+pkgname=pdsh
+version=2.33
+revision=1
+archs="x86_64 i686 aarch64 armv7l"
+wrksrc=pdsh-pdsh-${version}
```
[the remark this answers is not in the document]
Comment (knusbaum) [reply srcpkgs/pdsh/template]: Quoted.

**Comment by knusbaum** [comment]
@Chocimier Are there any other issues here?

**Pull request merged**